aw_watcher_terminal/config.py

```diff
@@ -1,26 +1,37 @@
 import argparse
+import logging
+from aw_core.log import setup_logging
 from aw_core.dirs import get_data_dir
 
 
-def load_config():
-    default_config = {
-        "pipe_path": "{}/{}".format(get_data_dir(("aw-watcher-terminal")),
-                                    "aw-watcher-terminal-fifo"),
-        "client_id": "aw-watcher-terminal-test-client",
-        "bucket_name": "aw-watcher-terminal",
-        "event_type": "app.terminal.activity",
-        "disabled": False,
-        "verbose": False,
-        "testing": False
-    }
+def load_config() -> None:
+    args = parse_args()
 
-    args = vars(parse_args())
-    config = {**default_config, **args}
+    global watcher_name
+    global data_dir
+    global client
+    global client_id
+    global bucket_id
+    global event_type
+    global logger
+    global verbose
+    global testing
 
-    return config
+    watcher_name = "aw-watcher-terminal"
+    data_dir = get_data_dir(watcher_name)
+    client = None
+    client_id = "{}-test-client".format(watcher_name)
```

Comment:
client_id should preferably be same as watcher_name

Reply:
Thanks for pointing it out, I've changed it in https://github.com/Otto-AA/aw-watcher-terminal/pull/5/commits/72703937048cb7bf4d3de7b87ac32c8ff396c1e1